### ISSUE TITLE
wall-mounted sparkers

### ISSUE THREAD

**Issue by NormalAnomaly** [BeeStation/BeeStation-Hornet#9943]
Issue reported from  Round ID: 46073 (BeeStation Sage - Roleplay)

Reporting client version: 514.1589

Joined, noticed this was still seeming to be an issue so getting it a proper report of some kind.

It seems that every map I have gotten to play so far since some very recent update, the toxins chambers have not had working igniters tied to their switches. Due to luck of the draw, I have not gotten to see if any of the maps with floor mounted igniters work, but all of the wall mounter 'sparker' igniters seem to be broken now. This ends up causing a need to enter the chamber and introduce a lighter or other ignition source.  I highly doubt this was intentional.

TL;DR: Seems every wall mounted igniter was broken with a recent change.

**Comment by Tsar-Salat** [comment]
Might've been #9865, I will check

**Comment by Tsar-Salat** [comment]
@NormalAnomaly rename the title of this Issue to "wall-mounted sparkers". Remove the "map issue", its a code issue. 

Fixing going up momentarily.
